scripts/feature-flag-audit.sh

```
@@ -0,0 +1,293 @@
+    echo -e "\n${BLUE}Checking flag metadata...${NC}\n"
+
+    # Check if metadata DB is available
+    local metadata_file="server/feature-flags/flag_metadata.json"
+
+    if [ ! -f "$metadata_file" ]; then
+        echo -e "${YELLOW}  No local flag metadata file found.${NC}"
+        echo "  Create $metadata_file to track flag ownership and expiration."
+        return 0
+    fi
+
+    local issues=0
```

Shellcheck (0.11.0)

[warning] issues appears unused. Verify use (or export if used externally).

(SC2034)